lib/srv/desktop/audit.go

```diff
@@ -47,11 +47,28 @@ type desktopSessionAuditor struct {
 	auditCache sharedDirectoryAuditCache
 }
 
+func (d *desktopSessionAuditor) makeSessionMetadata() events.SessionMetadata {
```

Comment:
Possible nit: why is this `makeSessionMetadata` while the similar function for `*ServerContext` is `GetSessionMetadata`?